core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -293,6 +293,27 @@ private[data] trait EitherTFunctions {
     def apply[E, A](opt: Option[A], ifNone: => E)(implicit F: Applicative[F]): EitherT[F, E, A] =
       EitherT(F.pure(Either.fromOption(opt, ifNone)))
   }
+
+  /**  If the condition is satisfied, return the given `A` in `Right`
+    *  lifted into the specified `Applicative`, otherwise, return the
+    *  given `E` in `Left` lifted into the specified `Applicative`.
+    *
+    * {{{
+    * scala> val userInput = "hello world"
```

Comment:
I had the doc example code running correctly, but the travis build was spitting out errors that the imports weren't needed. I actually went back and removed the imports just for the sake of appeasing Travis!

I'll push a new commit to see what happens. If it all goes well, I can squash before a merge.